src/components/Articles.jsx

```diff
@@ -0,0 +1,82 @@
+import {useEffect, useState} from 'react'
+import axios from 'axios';
+
+export default function Articles() {
+
+  const [articleList, setArticleList] = useState([])
+  const [pageNumber, setPageNumber] = useState(1)
+  const [totalArticles, setTotalArticles] = useState(0)
+  const [isFirstPage, setIsFirstPage] = useState(true)
+  const [isLastPage, setIsLastPage] = useState(false)
+
+  const pageLimit = Math.ceil(totalArticles / 10)
+
+  useEffect(() => {
+    axios.get(`https://news-api-so0z.onrender.com/api/articles?p=${pageNumber}`)
```

Comment:
You may want to extract fetching behaviour here to utils or api file. you'll have a lot of functions like this. Please use axios.create() too so you will have a base URL which can be used for all your request functions. (See NC Notes/Axios bottom of the page) ---> [https://notes.northcoders.com/courses/js-front-end/axios](url)